app/uploaders/profile_photo_attachment_uploader.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class ProfilePhotoAttachmentUploader < CarrierWave::Uploader::Base
```

Comment:
Should we perform any validation of the binary itself? Dimension restrictions, deeper format checks, virus checks?

Reply:
Frontend is enforcing some restrictions but of course somebody could invoke the API directly. 
@wunderhund raised the possibility of images as XSS attack vector, we need to think about that more carefully

Reply:
Is this a blocker for merging this for e2e testing? Or is it something we could work on in parallel?

Reply:
I think it's worth iterating on. I've added additional requirements to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/6979.